test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -67,13 +67,13 @@ if [ ! ${ZCC_TEST_NAME[$FRAMEWORK]} ]; then
     exit 1
 fi
 if [[ ${FRAMEWORK} == "tensorflow2" ]]; then
-    if [ "$(printf '%s\n' "0.7.2" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.7.2" ]; then
-        timeout 720s python mock/tests/zero_code_change/${FRAMEWORK}_gradtape_integration_tests.py 2>&1
+    if [ "$(printf '%s\n' "0.8.1" "$SMDEBUG_RELEASE_TAG" | sort -V | head -n1)" = "0.8.1" ]; then
```

Comment:
refer to this PR change https://github.com/aws/deep-learning-containers/pull/200/files#diff-6bcfd2946adc5ebde61805459f32293b.

Reply:
Could you also take a look into PR failure of time out issue?

Reply:
can you paste the codebuild link of the timeout?
@NihalHarish can you take a look at the timeout?

Reply:
If you check the build log of latest commit,  the ones in stage pending are the ones encounter time out issues. MXNet, TF1 and PyTorch could not pass currently.
There is a codebuild link for PyTorch
https://us-west-2.console.aws.amazon.com/codesuite/codebuild/projects/dlc-pr-sagemaker-test/build/dlc-pr-sagemaker-test%3Af7c67007-e172-4736-b68b-12fced391c38/log?region=us-west-2

Reply:
> refer to this PR change https://github.com/aws/deep-learning-containers/pull/200/files#diff-6bcfd2946adc5ebde61805459f32293b.

The testSmdebug file of your revision should be better. Should be able to update it after this turn of test run